test/integration/fake_upstream.cc

```diff
@@ -201,7 +201,7 @@ FakeUpstream::FakeUpstream(Ssl::ServerContext* ssl_ctx, uint32_t port,
 FakeUpstream::FakeUpstream(Ssl::ServerContext* ssl_ctx, Network::ListenSocketPtr&& listen_socket,
                            FakeHttpConnection::Type type)
     : ssl_ctx_(ssl_ctx), socket_(std::move(listen_socket)),
-      handler_(stats_store_, log(), Api::ApiPtr{new Api::Impl(std::chrono::milliseconds(10000))}),
+      handler_(log(), Api::ApiPtr{new Api::Impl(std::chrono::milliseconds(10000))}),
```

Comment:
The stats_store_ was only passed through this call chain because it was used by the old watchdog timer.  Moving the stats incrementing activity into GuardDog made these members unused/args not needed, so I've removed them.